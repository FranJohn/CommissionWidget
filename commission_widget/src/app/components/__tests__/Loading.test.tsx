import '@testing-library/jest-dom';
import React from 'react';
import { render } from '@testing-library/react';
import LoadingSpinner from '../Loading';

describe('LoadingSpinner Component', () => {
    it('renders the LoadingSpinner component', () => {
        const { container } = render(<LoadingSpinner />);
        const loadingSpinner = container.querySelector('.loading-spinner');
        expect(loadingSpinner).toBeInTheDocument();
    });

    it('matches the LoadingSpinner snapshot', () => {
        const { container } = render(<LoadingSpinner />);
        expect(container).toMatchSnapshot();
    });
});
