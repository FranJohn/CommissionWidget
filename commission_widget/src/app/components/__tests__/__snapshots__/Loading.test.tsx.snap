// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner Component matches the LoadingSpinner snapshot 1`] = `
<div>
  <div
    class="loading-spinner"
    style="width: 50px; height: 50px;"
  >
    <div
      class="spinner"
    />
  </div>
</div>
`;
